### DISCUSSION TITLE
Can I use xterm as a tree editor?

### DISCUSSION THREAD

**Discussion by ivan-suhorukov** [sourcelair/xterm.js#5133] [Q&A] [answered]
Hello! I'm writing a tool for editing data trees of json, yaml, XML and so on in one visual way. That's not a text editor, it's data editor. And I need a graphical area with ability to show a text in ways I want.

How do you suggests, does xterm.js itself, without any underlying terminal program, satisfying these needs?

Or may be there is a better way?

Thanks!

**Comment by InTEGr8or** [comment, marked as answer]
I wonder if it might not be better to use the Monaco editor for this.

That's what I'm going to do after seeing this is one of the only mentions of this on the web. I am going to try to implement a tree-view file selector in Monaco that I could only get to display in xterm.js.
